### DISCUSSION TITLE
How to get movie set art work to download?

### DISCUSSION THREAD

**Discussion by jmayer004** [Komet/MediaElch#1444] [Q&A]
Anyone get movie set art work to download?

**Comment by bugwelle** [comment]
The movie set feature is not feature complete. Some features are missing, for example artwork download is limited. If you have selected a movie set, you can click on the right poster image to download images.

**Reply by jmayer004** [reply]
Thanks for responding. I thought I was to blame after hours trying to get it working. My time is irrelevant. Still a great program. I even donated. Keep up with the good work. Congrats on new job and such.....

**Reply by bugwelle** [reply]
> My time is irrelevant.

No, it's not. If MediaElch made you search for some settings for hours, MediaElch is to blame (or at least its documentation).

> Still a great program. I even donated. 

Thanks! Donations go to Daniel ("Komet"), i.e. the original MediaElch author. :+1:

> Keep up with the good work. Congrats on new job and such.....

Thank you very much! :-)

**Reply by jmayer004** [reply]
I scrape everything to my local folder (movie (date)). Tried assigning a folder, it assigns but no artwork (poster) for set. Tried next to movie, nope, not working. Work around, once scrapped into Kodi refresh info from TMDB It imports info to Kodi data base not movie folders. I'm polish and make mistakes, old to. That is why I opted to scrape to local folders.

**Reply by jmayer004** [reply]
But thanks
